HW04/vitaly_kadyrko/lib/homework.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Homework
+  attr_accessor :status
+  attr_reader :title, :description, :student, :mentor, :answers
+
+  def initialize(title:, description:, student:, mentor:, status: 'Created')
+    @title = title
+    @description = description
+    @student = student
+    @mentor = mentor
+    @status = status
+    @answers = []
+  end
+
+  def notify_mentor(notification)
```

Comment:
Does homework need to notify students and mentors? Or is it students' and mentors' responsibility?

Reply:
I thought it would be a good idea if the homework would be an intermediary between the mentor and the student. And these two methods will be in the same class.

Reply:
But I will change)